lib/ipinfo/adapter.rb

```diff
@@ -33,10 +33,15 @@ def connection(adapter)
     end
 
     def default_headers
-        {
-          'User-Agent' => "ipinfo-ruby/#{::IpinfoIo::VERSION}",
+        headers = {
+          'User-Agent' => "IPinfoClient/Ruby/1.0",
           'Accept' => 'application/json'
         }
+        if token
+          headers['Authorization'] = "Bearer #{CGI::escape(token)}"
+        end
+        puts headers
```

Comment:
Would be great to not have a debugging code in a library, nobody likes libraries spitting out tons of debugging info without ability to turn it off.